dojo/tools/awssecurityhub/compliance.py

```diff
@@ -7,13 +7,21 @@ def get_item(self, finding: dict, test):
         finding_id = finding.get("Id", "")
         title = finding.get("Title", "")
         severity = finding.get("Severity", {}).get("Label", "INFORMATIONAL").title()
+        for resource_id_arn in finding.get("Resources", []):  # adding a generic resource_id implementation separate from the below AwsEcrContainerImage one
+            description_resource_id_arn_value = resource_id_arn.get("Id")
         mitigation = ""
         impact = []
         references = []
         unsaved_vulnerability_ids = []
         epss_score = None
         mitigation = finding.get("Remediation", {}).get("Recommendation", {}).get("Text", "")
+        mitigation += finding.get("Remediation", {}).get("Recommendation", {}).get("Url", "")
         description = "This is a Security Hub Finding \n" + finding.get("Description", "")
+        description += f"\n**AWS Finding ARN:** {finding_id}\n"
```

Comment:
Since none of the other fields are bolded, we should remove it here to be consistent.

```suggestion
        description += f"\nAWS Finding ARN: {finding_id}\n"
```